### ISSUE TITLE
Add HTTP authentication to options

### ISSUE THREAD

**Issue by frederickjh** [raviqqe/muffet#48]
Currently Muffet has no way to supply the username and password for a HTPASSWD protected website.

This feature request is to add  username and password options that would allow Muffet to login to a site to check it.

**Comment by raviqqe** [comment]
Isn't `--header` option sufficient for you?

**Comment by frederickjh** [comment]
I have googled around how to send the username and password via a header, but having to base64 encode the username and password is a real pain. However it does work

Here is what worked:
[base64 encode](https://www.base64encode.org/) the username and password separated by only a colon(:). I used **username:password** for this example. Which returned: **dXNlcm5hbWU6cGFzc3dvcmQ=**.

So, in a muffet command:
```bash
muffet -j "Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=" http://example.com
```
This works to authenticate but I will leave it up to you as to whether you want to implement HTTP authentication or just close this issue.
